src/methods/spectral_partial_randomization.jl

```diff
@@ -0,0 +1,95 @@
+export SpectralPartialRandomization, SpectralPartialRandomizationAAFT
+
+"""
+    SpectralSpectralPartialRandomization(α = 0.5)
```

Comment:
Same as above: please leave a blank line between the type signature and the rest of the docstring.